recipes/json-c/meta.yaml

```diff
@@ -0,0 +1,35 @@
+package:
+    name: json-c
+    version: 0.12
+
+source:
+    fn: json-c-0.12.tar.gz
+    url: https://s3.amazonaws.com/json-c_releases/releases/json-c-0.12.tar.gz
+    md5: 3ca4bbb881dfc4017e8021b5e0a8c491
+
+build:
+    number: 0
+    skip: True  # [win]
+
+requirements:
+    build:
+        - autoconf
+        - automake
+        - libtool
+    run:
+        - libgcc
+
+test:
+    commands:
+        - test -f ${PREFIX}/lib/libjson-c.a  # [unix]
+        - test -f ${PREFIX}/lib/libjson-c.so  # [linux]
+        - test -f ${PREFIX}/lib/libjson-c.dylib  # [osx]
+
+about:
+    home: https://github.com/json-c/json-c
```

Comment:
They [mention]( https://github.com/json-c/json-c#building-on-unix-with-git-gcc-and-autotools ) the GitHub wiki as the homepage. Should we care? Not a big deal, but figured it should be mentioned.